cli/command/formatter/formatter.go

```diff
@@ -7,17 +7,21 @@ import (
 	"text/tabwriter"
 	"text/template"
 
-	"github.com/docker/cli/templates"
 	"github.com/pkg/errors"
+
+	"github.com/docker/cli/templates"
```

Comment:
Looks like this was accidentally changed

Reply:
Fixed